### PR TITLE
Bug/e UI 6877

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-6877

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]
LGTM

File: config/default.json [hunk 1]
```diff
@@ -69,7 +69,8 @@
     },
     "refunds": "http://ccpay-refunds-api-prod.service.core-compute-prod.internal",
     "role_assignment": {
-      "roleApi": "http://am-role-assignment-service-prod.service.core-compute-prod.internal"
+      "roleApi": "http://am-role-assignment-service-prod.service.core-compute-prod.internal",
+      "roleMappingApi": "http://am-org-role-mapping-service-demo.service.core-compute-demo.internal"
```
Comment (connorpgpmcelroy) [inline config/default.json:73]: demo url in default.json needs to be changed to prod presumably?

File: package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "test:coverage": "yarn test:coverage:ng",
     "test:coverage:all": "yarn test:coverage:ng && yarn test:coverage:node",
     "test:coverage:merge": "./node_modules/.bin/lcov-result-merger './reports/tests/coverage/*/lcov.info' > ./reports/tests/coverage/lcov.info",
-    "test:coverage:node": "cd api && yarn coverage",
+    "test:coverage:node": "cd api && yarn ccoverage",
```
Comment (connorpgpmcelroy) [inline package.json:34]: ccoverage?

File: src/work-allocation/components/task-list-filter/task-list-filter.component.ts [hunk 3]
```diff
@@ -84,7 +84,7 @@ export class TaskListFilterComponent implements OnInit, OnDestroy {
     if (this.router.getCurrentNavigation() &&
       this.router.getCurrentNavigation().extras.state &&
       this.router.getCurrentNavigation().extras.state.location) {
-      this.bookingLocations = this.router.getCurrentNavigation().extras.state.location.ids;
+      this.bookingLocations = [this.router.getCurrentNavigation().extras.state.location.id];
```
Comment (connorpgpmcelroy) [inline src/work-allocation/components/task-list-filter/task-list-filter.component.ts:87]: concerned about this - does this mean multiple locations are now not being returned from the booking screens to the my work filter?

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Have two requested changes and a query

File: src/work-allocation/components/task-list-filter/task-list-filter.component.ts (hunk 3 above)
Comment (MateiHMCTS) [reply src/work-allocation/components/task-list-filter/task-list-filter.component.ts]: Found one place where ids was used, so now I have changed all the other to pass an array

Comment (udaydenduluri33) [reply src/work-allocation/components/task-list-filter/task-list-filter.component.ts]: [@Matei Baciu](https://hmcts-reform.slack.com/team/U03MLGQQ9RQ)
 can you liaise with 
[@Sreekanth Puligadda](https://hmcts-reform.slack.com/team/UL2H260TT)
 on the comment made by 
[@Connor McElroy](https://hmcts-reform.slack.com/team/U01DL3S8HB5)
 and look for different use cases?

File: package.json (hunk 2 above)
Comment (udaydenduluri33) [reply package.json]: Looks like a spell check

File: config/default.json (hunk 1 above)
Comment (udaydenduluri33) [reply config/default.json]: @MateiHMCTS  URL in prod always need to be pointing to PROD
You need to change it in your local-development or local-demo

**Review by udaydenduluri33 — CHANGES_REQUESTED** [review]
Please review comments by @connorpgpmcelroy

File: src/work-allocation/components/task-list-filter/task-list-filter.component.ts (hunk 3 above)
Comment (MateiHMCTS) [reply src/work-allocation/components/task-list-filter/task-list-filter.component.ts]: Sure, will do

**Review by udaydenduluri33 — APPROVED** [review]

File: config/default.json [hunk 4]
```diff
@@ -69,7 +69,8 @@
     },
     "refunds": "http://ccpay-refunds-api-prod.service.core-compute-prod.internal",
     "role_assignment": {
-      "roleApi": "http://am-role-assignment-service-prod.service.core-compute-prod.internal"
+      "roleApi": "http://am-role-assignment-service-prod.service.core-compute-prod.internal",
+      "roleMappingApi": "http://am-org-role-mapping-service-prod.service.core-compute-demo.internal"
```
Comment (connorpgpmcelroy) [inline config/default.json:73]: still says compute-demo

**Review by connorpgpmcelroy — APPROVED** [review]
Approving on bases that  default .json comment addressed

**Pull request merged**